.gitignore

```diff
@@ -1,2 +1,4 @@
 target
 work
+/github-branch-source.iml
```

Comment:
You and new contributors will change gitignore every time and it will be infinitely. Better add all required at once ;)